pkg/scripts/os.go

```diff
@@ -57,9 +56,9 @@ sudo DEBIAN_FRONTEND=noninteractive apt-get install --option "Dpkg::Options::=--
 
 {{- if .CONFIGURE_REPOSITORIES }}
 curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
-curl -fsSL https://download.docker.com/linux/${ID}/gpg | sudo apt-key add -
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
 
-echo "deb https://download.docker.com/linux/${ID} $(lsb_release -sc) stable" |
+echo "deb https://download.docker.com/linux/ubuntu bionic stable" |
```

Comment:
Can we add a TODO to replace this once 1.16 is not supported anymore?

Reply:
👍